modules/standard/ChapelIO.chpl

```diff
@@ -834,12 +834,17 @@ module ChapelIO {
   }
 
   /* Equivalent to ``try! stdout.write``. See :proc:`IO.channel.write` */
-  proc write(const args ...?n) {
-    try! stdout.write((...args));
+  proc write(const args ...?n):bool{
+    try! {
+      return stdout.write((...args));
+    }
```

Comment:
I suggest we do not need these extra curly braces.
What is the benefit of adding them?

Reply:
I believe the curl braces make the procedure more readable other than that I do not believe they do matters.